extensions/mssql/package.nls.json

```diff
@@ -183,8 +183,8 @@
 	"mssql.objectExplorer.groupBySchema": "When enabled, the database objects in Object Explorer will be categorized by schema.",
 	"mssql.objectExplorer.enableGroupBySchema": "Enable Group By Schema",
 	"mssql.objectExplorer.disableGroupBySchema": "Disable Group By Schema",
-	"mssql.objectExplorer.enableGroupBySchemaTitle": "SQL Server: Enable Group By Schema",
-	"mssql.objectExplorer.disableGroupBySchemaTitle": "SQL Server: Disable Group By Schema",
+	"mssql.objectExplorer.enableGroupBySchemaTitle": "Enable Group By Schema",
```

Comment:
We should just remove these "duplicate" commands - IIRC they were added specifically because we wanted this different text but they end up routing to the same command functionality in the end.

Reply:
Look like duplicates to me : https://github.com/microsoft/azuredatastudio/blob/25cdb558a707b55ada24b58489b9ad387dbde42d/extensions/mssql/src/main.ts#L105

Reply:
Actually - isn't that why we wanted them, for the mouseover text on the toolbar?

![image](https://github.com/microsoft/azuredatastudio/assets/28519865/930b83ea-be59-4ef6-a7a7-6bc549e75bb1)

So we should probably keep that to avoid confusion (since it only applies to MSSQL items) and just switch the commands so this is the hidden one enableGroupBySchema is what shows in the command palette.

Reply:
Addressed + screenshot updated in PR description.